www/src/components/sidebar/section-title.js

```diff
@@ -165,7 +165,7 @@ const SplitButton = withI18n()(({ i18n, itemRef, item, uid }) => {
       </span>
       <button
         aria-controls={uid}
-        aria-expanded={isExpanded}
+        aria-expanded={!!isExpanded}
```

Comment:
There's one more aria-expanded in `sectionTitle` that could use this.

Alternately, in the line above where `isExpanded` is called you can do

```js
// used to set `aria-expanded`, so needs to be a boolean value.
const { isExpanded = false } = getItemState(item)
```

Reply:
Sure, added `{ isExpanded = false } ` to lines 40, 90, and 137 of `section-title`.